lib/chat/db/supervision.livemd

```diff
@@ -211,7 +211,7 @@ graph TD;
 `Mounter` mounts device into folder specified. And unmounts when terminated.
 
 `Decider` figures out which functionality to start under `FunctionalityDynamicSupervisor`.
-It does it by checking whether device contains appropriate directory (`main_db` for backup, `cargo_db` for cargo, or `onliners_db` for onliners sync).
+It does it by checking whether device contains appropriate directory (`backup_db` for backup, `cargo_db` for cargo, or `onliners_db` for onliners sync).
```

Comment:
we have AdminDB document where we describe structure of records. Please update it as well